vizro-core/tools/update_material_icons.py

```diff
@@ -0,0 +1,31 @@
+"""Script to update the Google Material Design Icons font in the static folder."""
+
+import logging
+from pathlib import Path
+
+import requests
+
+FONT_URL = "https://github.com/google/material-design-icons/raw/refs/heads/master/variablefont/MaterialSymbolsOutlined%5BFILL,GRAD,opsz,wght%5D.woff2"
+LOCAL_PATH = Path(__file__).parent.parent / "src/vizro/static/css/fonts/material-symbols-outlined.woff2"
+STATUS_OK = 200
+TIMEOUT = 10  # seconds
+
+logging.basicConfig(level=logging.INFO, format="%(message)s")
+logger = logging.getLogger(__name__)
+
+
+def download_font(url, path):
+    """Downloads the Material Design Icons font from Google Fonts and saves it to the local static folder."""
+    try:
+        response = requests.get(url, timeout=TIMEOUT)
+        if response.status_code == STATUS_OK:
```

Comment:
```suggestion
        if response.status_code == 200:
```
Will it ever change?

Reply:
No, but I changed this because, otherwise the linter complains that it's a magic number and it wants it stored in a variable 😄

Reply:
lol, ok fair enough